docs/vereisten/verplichtingen_van_aanbieders_van_ai_modellen_voor_algemene_doeleinden.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Verplichtingen van aanbieders van AI-modellen voor algemene doeleinden 
+toelichting: Aanbieders van AI-modellen voor algemene doeleinden moeten (technische) informatie en documentatie opstellen, up-to-date houden en beschikbaar stellen voor aanbieders van AI-systemen die het AI-model voor algemene doeleinden in hun AI-systemen willen integreren. 
+status_vereiste:
+- nog-niet-geldend
+levenscyclus:
+- ontwerp
+- dataverkenning-en-datapreparatie
+- ontwikkelen
+- verificatie-en-validatie
+- implementatie
+- monitoring-en-beheer
+- uitfaseren
+bouwblok:
+- transparantie
+hide:
+- navigation
+---
+
+<!-- tags -->
+## Vereiste
+
+Aanbieders van AI-modellen voor algemene doeleinden moeten (technische) informatie en documentatie opstellen, up-to-date houden en beschikbaar stellen voor aanbieders van AI-systemen die het AI-model voor algemene doeleinden in hun AI-systemen willen integreren.
+
+
+## Toelichting
+
+Aanbieders van AI-modellen voor algemene doeleinden hebben een bijzondere rol en verantwoordelijkheid binnen de AI-waardeketen.
+Zij leveren modellen die de basis kunnen vormen voor een reeks systemen verder in de AI-waardeketen, veelal aangeboden door aanbieders verder in de AI-waardeketen.
+Dit vraagt om een goed inzicht in de modellen en hun capaciteiten, zowel qua integratie van de modellen in hun producten als qua naleving van hun verplichtingen.
+
+
+Er zijn daarom evenredige transparantiemaatregelen nodig, zoals opstelling en bijwerking van documentatie en verstrekking van informatie over het AI-model voor algemeen gebruik door de aanbieders verder in de AI-waardeketen.
+De aanbieder van het AI-model voor algemene doeleinden dient technische documentatie op te stellen en bij te werken om deze op verzoek te kunnen overleggen aan het AI-bureau en de nationale bevoegde autoriteiten.
```

Comment:
Wordt met AI-bureau hier het AI-office bedoelt wat er vanuit de EU gaat komen? Is dat al een gangbare Nederlandse term, en willen we daar al naar verwijzen?